Makefile

```diff
@@ -4,8 +4,12 @@
 NPM_VERSION := $(shell npm --version 2>/dev/null)
 NODE_VERSION := $(shell node --version 2>/dev/null)
 GO_VERSION := $(shell go version 2>/dev/null)
+CARGO_VERSION := $(shell cargo --version 2>/dev/null)
+GIT_VERSION := $(shell git --version 2>/dev/null)
 AXOLOTL_GIT_VERSION := $(shell git tag | tail --lines=1)
 AXOLOTL_VERSION := $(subst v,,$(AXOLOTL_GIT_VERSION))
+uname_p := $(shell uname -p)
```

Comment:
What about also uppercasing these two variables? Then it aligns well with the other ones.

Reply:
I thought about it, too and I agree, it's like that, because it was part of a copy+paste snippet.